src/pages/[username].js

```diff
@@ -0,0 +1,20 @@
+import { useRouter } from "next/router";
+import { useEffect, useState } from "react";
+import App from './index'
+
+export default function PrefilledUsername() {
+
+  const route = useRouter().asPath
+  const [usernameProps, setUsernameProps] = useState("");
```

Comment:
```suggestion
  const [username, setUsername] = useState("");
```
let's not add `props` in variable, make it simpler.